libraries/Microsoft.Bot.Builder.AI.LuisV3/LuisPredictionOptions.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.AI.Luis
+{
+    /// <summary>
+    /// Optional parameters for a LUIS prediction request.
+    /// </summary>
+    public class LuisPredictionOptions
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether all intents come back or only the top one.
+        /// </summary>
+        /// <value>
+        /// True for returning all intents.
+        /// </value>
+        public bool IncludeAllIntents { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not instance data should be included in response.
+        /// </summary>
+        /// <value>
+        /// A value indicating whether or not instance data should be included in response.
+        /// </value>
+        public bool IncludeInstanceData { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether API results should be included.
+        /// </summary>
+        /// <value>True to include API results.</value>
+        /// <remarks>This is mainly useful for testing or getting access to LUIS features not yet in the SDK.</remarks>
+        public bool IncludeAPIResults { get; set; } = false;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether queries should be logged in LUIS.
+        /// </summary>
+        /// <value>
+        /// If queries should be logged in LUIS in order to help build better models through active learning.
+        /// </value>
+        public bool Log { get; set; } = true;
```

Comment:
The default value is false in LUIS for privacy reasons. Setting this as true is a consent from the user that they agree for their data to get outside the region they are using. So I think this should be the case for the SDK as well.

Reply:
What I've done is make log nullable with null corresponding to the default on the LUIS side.  I sent an email, but I think we should ask the user when they create an app if they want to use active learning and set the log flag per app based on that.